### ISSUE TITLE
(not important) session __str__ typo

### ISSUE THREAD

**Issue by uukelele-scratch** [TimMcCool/scratchattach#300]
calling print() on a session always says "Login for account: {self.username}" without the actual username.

Relevant part of code (for class Session(BaseSiteComponent)): 

```
   def __str__(self):
        return "Login for account: {self.username}"

```

Easy fix: use an f-string (I think you forgot to put it in)

**Comment by FAReTek1** [comment]
yeah i think i fixed that in one of my 5 prs but i dont remember which or if I'm misremembering

**Comment by FAReTek1** [comment]
its fixed in #294

**Comment by TimMcCool** [comment]
fixed
